Gems/ROS2/Code/Source/RobotImporter/RobotImporterWidget.cpp

```diff
@@ -182,7 +182,7 @@ namespace ROS2
             auto collidersNames = Utils::GetMeshesFilenames(m_parsedUrdf->getRoot(), false, true);
             auto visualNames = Utils::GetMeshesFilenames(m_parsedUrdf->getRoot(), true, false);
 
-            AZStd::string_view dirSuffix = "";
+            AZStd::string dirSuffix = "";
```

Comment:
Line 195 below needs to be changed to create an `AZStd::string` or alternatively use the ToFixedString function.
```suggestion
            AZ::Uuid::FixedString dirSuffix;
            // ...
            dirSuffix = paramsUuid.ToFixedString();
```